.gitignore

```diff
@@ -7,3 +7,5 @@
 tmp
 
 \.vscode/
+
+runsc\.sha512
```

Comment:
why not `wget` this file into `tmp` so we don't need to have it lying around in root?